gir/pkgconfig/pkgconfig_test.go

```diff
@@ -2,42 +2,77 @@ package pkgconfig
 
 import (
 	"fmt"
-	"reflect"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Could you switch `testify` out for `alecthomas/assert/v2`? It should provide both `assert.{Equal,NoError}` .

Reply:
Will do.